hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/handshakeserver/ConnectionHandler.java

```diff
@@ -190,25 +190,22 @@ private void performTransaction(BufferedInputStream socketInput, BufferedOutputS
 
 		logger.info("{} - TransactionId: {} Start performing message transaction: {}", methodName, transactionId,  ret_code);
 		// read SI segment
-		if (socketInput.available() > 0) {
-			byte[] message = new byte[SI_MESSAGE_LENGTH];
-			socketInput.read(message);
-			
-			HandshakeUtil.unScrambleData(message, decodeSeed);
-			logInputData(message);					
-			logger.debug("{} - TransactionId: {}  Received from originator {} byte SI Data Block: {}", methodName, transactionId,  message.length,
+		
+		logger.info("{} - TransactionId: {} Start reading SI segment: {}", methodName, transactionId,  ret_code);
```

Comment:
Can we make this debug as well? It's similar to the other debug statements in that it's quite detailed.